traffic_portal/app/src/common/modules/table/servers/TableServersController.js

```diff
@@ -17,405 +17,634 @@
-                clearCDNServerUpdates(cdn.id);
-            }, function () {
-                // do nothing
-            });
-
-
-        } else {
-            params = {
-                title: 'Clear Server Updates',
-                message: "Please select a CDN"
-            };
-            var modalInstance = $uibModal.open({
-                templateUrl: 'common/modules/dialog/select/dialog.select.tpl.html',
-                controller: 'DialogSelectController',
-                size: 'md',
-                resolve: {
-                    params: function () {
-                        return params;
-                    },
-                    collection: function(cdnService) {
-                        return cdnService.getCDNs();
-                    }
-                }
-            });
-            modalInstance.result.then(function(cdn) {
-                clearCDNServerUpdates(cdn.id);
-            }, function () {
-                // do nothing
-            });
-        }
-    };
-
-    $scope.refresh = function() {
-        $state.reload(); // reloads all the resolves for the view
-    };
-
-    $scope.toggleVisibility = function(colName) {
-        const col = serversTable.column(colName + ':name');
-        col.visible(!col.visible());
-        serversTable.rows().invalidate().draw();
-    };
-
-    $scope.ssh = serverUtils.ssh;
-
-    $scope.isOffline = serverUtils.isOffline;
-
-    $scope.offlineReason = serverUtils.offlineReason;
-
-    $scope.getRelativeTime = dateUtils.getRelativeTime;
-
-    $scope.navigateToPath = locationUtils.navigateToPath;
-
-    var init = function () {
-        getStatuses();
-    };
-    init();
-
-    angular.element(document).ready(function () {
-        serversTable = $('#serversTable').DataTable({
-            "lengthMenu": [[25, 50, 100, -1], [25, 50, 100, "All"]],
-            "iDisplayLength": 25,
-            "aaSorting": [],
-            "columns": $scope.columns,
-            "initComplete": function(settings, json) {
-                try {
-                    // need to create the show/hide column checkboxes and bind to the current visibility
-                    $scope.columns = JSON.parse(localStorage.getItem('DataTables_serversTable_/')).columns;
-                } catch (e) {
-                    console.error("Failure to retrieve required column info from localStorage (key=DataTables_serversTable_/):", e);
-                }
-            }
-        });
-    });
+var TableServersController = function(servers, $scope, $state, $uibModal, $window, dateUtils, locationUtils, serverUtils, cdnService, serverService, statusService, propertiesModel, messageModel, userModel, $document) {
+	/**** Table cell formatters/renderers ****/
 
+	// browserify can't handle classes...
+	function SSHCellRenderer() {}
+	SSHCellRenderer.prototype.init = function(params) {
+		this.eGui = document.createElement("A");
+		this.eGui.href = "ssh://" + userModel.user.username + "@" + params.value;
```

Comment:
not sure if you want to / can address this as part of this PR:

https://github.com/apache/trafficcontrol/issues/1801

Reply:
I could - and considered it - but I don't want to. For this PR I just want to make sure the new table(s) has/have all the same functionality as was expected of the previous version.